dataactbroker/fileRoutes.py

```diff
@@ -27,6 +27,32 @@ def add_file_routes(app, create_credentials, is_local, server_path):
     @requires_login
     def submit_files():
         file_manager = FileHandler(request, is_local=is_local, server_path=server_path)
+
+        sess = GlobalDB.db().session
+
+        start_date = request.json.get('reporting_period_start_date')
```

Comment:
All three routes are similar, but there are slightly different params we can use.  For example, `/submit_files/` stores the state values of cgac_code, reporting_start_date, reporting_end_date, but these are not supplied when we try to update files for certified submission. We might want to reconsider adding a check on `submit_files` since we already have a check when users try to certify submission

I changed the `/v1/check_year_quarter/` to utilize `kwargs`. For the `certify_submission/` there really no benefit of using `kwargs`, adding kwargs also makes the code harder to work with. In my opinion, it's best to keep it as it is. 

I did, however, change the return error to function to avoid repeated code. Although that again not really best practice in my opinion since all three routes should be able to return different messages. It will be hard to work with in the future if we combine 3 return errors in to one. Again, I will leave it up to you @nmonga91.